src/Controller/FindByPid/GalleryController.php

```diff
@@ -62,4 +68,14 @@ public function getFirstImage(?string $imagePid, array $images): ?Image
         }
         return $image;
     }
+
+    public function getSchema(StructuredDataHelper $structuredDataHelper, Array $images, Programme $programme, Gallery $gallery)
```

Comment:
```suggestion
    public function getSchema(StructuredDataHelper $structuredDataHelper, array $images, Programme $programme, Gallery $gallery)
```
I don't think this breaks anything but my understanding is that `array` should be lowercase

Reply:
I think this could have been missed